framework/src/problems/FEProblemBase.C

```diff
@@ -2695,62 +2693,17 @@ FEProblemBase::addFVKernel(const std::string & fv_kernel_name,
                            const std::string & name,
                            InputParameters & parameters)
 {
-  if (_displaced_problem && parameters.get<bool>("use_displaced_mesh"))
-  {
-    parameters.set<SubProblem *>("_subproblem") = _displaced_problem.get();
-    parameters.set<SystemBase *>("_sys") = &_displaced_problem->nlSys();
-  }
-  else
-  {
-
-    // TODO: this if clause was copied from DG Kernels - do we really need it for FV?
-    if (_displaced_problem == nullptr && parameters.get<bool>("use_displaced_mesh"))
-    {
-      if (parameters.have_parameter<bool>("use_displaced_mesh"))
-        parameters.set<bool>("use_displaced_mesh") = false;
-    }
-
-    parameters.set<SubProblem *>("_subproblem") = this;
-    parameters.set<SystemBase *>("_sys") = _nl.get();
-  }
-
-  for (THREAD_ID tid = 0; tid < libMesh::n_threads(); ++tid)
-  {
-    std::shared_ptr<FVKernel> k = _factory.create<FVKernel>(fv_kernel_name, name, parameters, tid);
-    theWarehouse().add(k);
-  }
+  // TODO: this changes an overridden use_displaced_mesh - do we really need it for FV?
```

Comment:
@rwcarlsen @lindsayad What do you say to this?

Reply:
This was in there before - I just lazily copy-pasta. 

We do it for standard Kernels too, so I'd say it should probably be there, too?

Reply:
I think this looks fine.

Reply:
So - good to remove the TODO comment, then?

Reply:
Nice. Yea!